src/constants/tooltips.js

```diff
@@ -61,15 +61,15 @@ export default {
     apiKey: (coin) => {
       if (coin === 'eth' || coin === 'token') {
         return 'An Api-Key Token from etherscan.com required for Ethereum Mainnet recoveries';
-      } else if (coin === 'btc') {
-        return 'An Api-Key Token from blockchair.com required for Bitcoin Mainnet and Testnet recoveries';
+      } else if (coin === 'bsv') {
```

Comment:
why did we remove the BTC tooltip?

Reply:
we no longer need it for btc, because we are using blockstream for btc now :)